### ISSUE TITLE
Internationalization for script mod result in mod issue detection.

### ISSUE THREAD

**Issue by BR146** [lvgamedev/simairport-modding#103]
When adding internationalization to a script mod's name and description, the game shows the "Mod Issue Detected" message when loading a game (including a new savegame).

Game Version: edge 22.05.2020 12:00:00

### Mod Class without i18n
Assume starting with the following mandatory BaseMod derivate:
```
namespace BR146.ModTemplate {
    public class Mod : BaseMod {

        public override string Name => "BR146 ModTemplate Name";
        public override string Description => "BR146 ModTemplate Desc";

        // The other required stuff omitted for clarity
    }
}
```
This loads fine into a new airport, no mod issue is detected.
As reference the log produced by this implementation: [log no i18n.log](https://github.com/lvgamedev/simairport-modding/files/4671953/log.no.i18n.log)

### Mod Class with i18n
Changing up the Name and Description property to use i18n:
```
namespace BR146.ModTemplate {
    public class Mod : BaseMod {

        public override string Name => i18n.Get("BR146.ModTemplate.name");
        public override string Description => i18n.Get("BR146.ModTemplate.desc");

        // The other required stuff omitted for clarity
    }
}
```
and the language file _em.xml_:
```<?xml version="1.0" encoding="utf-8" ?>
<BR146>
	<ModTemplate>
		<name>ModTemplate Name</name>
		<desc>ModTemplate Description</desc>
	</ModTemplate>
</BR146>
```

The translation is successfully shown in the Mod Settings in the main menu. Therefore I conclude the above mentioned code is correct and working.
![ModSettings_MainMenu](https://user-images.githubusercontent.com/65818738/82733173-6d7b7100-9d12-11ea-9b6e-c4d26ad7f3fe.png)

Loading/Starting an airport, the "Mod Issue Detected" message pops up. The mod settings accessible by the escape menu still shows the correct translation:
![ModSettings_PauseMenu](https://user-images.githubusercontent.com/65818738/82733216-a3205a00-9d12-11ea-86fc-3bb88196ed61.png)

The following log is produced: [log with i18n.log](https://github.com/lvgamedev/simairport-modding/files/4671955/log.with.i18n.log)

Note: I have a few mods currently active. But as the mod issue only shows up after changing the aforementioned two lines, these mods are most likely not the culprits.

### Log Comparison

Comparing both logs these two additional lines are logged when i18n is used:
1. `[0]: Mod "BR146.ModTemplate.dll" was found twice, only loading local version!`
2. `[0]: Mod issue detected, notifying player ...`

I ignored obvious log differences in time and execution times as well as the actual mod name (as this obviously differ because of the implementation). The logs are attached above, so you can check yourself ;)

The second entry is of no surprise, as the mod issue screen is detected.
The first entry hints at some issue loading the mod twice? This is highly likely to cause the mod issue detection in the first place?

### Thoughts
Internationalization of a script mod's name and description causes the game to load the mod a second time, which is detected and issued to the user. This may be an issue of using the mod's "readable" name and not the mod's internal name as lookup?

**Comment by joscha999** [comment]
Not related to i18n - i18n would show a different (or as of currently) no issue.
The mod issue detected flag is set because the dll was found twice. Make sure to a) restart the game between changes of the mod to actually reload the assembly and b) not be subscribed to your mod while wokring on it locally!

**Comment by BR146** [comment]
I just reproduced it again using this example mod I just created:
[Example Code.zip](https://github.com/lvgamedev/simairport-modding/files/4808640/Example.Code.zip)
[Example Mod.zip](https://github.com/lvgamedev/simairport-modding/files/4808639/Example.Mod.zip)
It is not uploaded nor is it added twice. Simairport was closed while adding this mod.

Reproduce:
1. No mods installed, Simairport closed.
2. Add example mod to Mod folder
3. Start simairport
4. Start new airport -> "Mod issue detected".

Step 4 is optional. The log file already contains the "loaded twice" notice at step 3.

Log file output after Step 3 (and closing game):
```
[0]: Logger set up. DateTime: 06/21/2020 00:11:14
[0]: Mod "BR146.I18N_Issue.dll" was found twice, only loading local version!
[0]: Loaded mod "" from "BR146"
[0] C:\Repositories\simairport\Assets\Main Menu\MainMenu_UI.cs|Start|39: Game Version: edge 09.06.2020 12:30:00
[0]: FileLogger shutting down - Log.InstanceCount: 1
```

Seems like the translation triggers a mod reload directly at startup? Also notice the missing translation.
